modules/Microsoft.HealthDataServices/readme.md

```diff
@@ -0,0 +1,87 @@
+# DICOM Services `[Microsoft.HealthcareApis/workspaces]`
```

Comment:
Great work with the Readme. BUT - in case you were not aware, you can use the utility [`Set-ModuleReadme`](https://github.com/Azure/ResourceModules/blob/main/utilities/tools/Set-ModuleReadMe.ps1) to generate almost all of the content for a template for you.

Please apply it to this and the other modules every time to are done changing a template. Some sections are manual and would need to be updated, but done once, they stay as is.

Also note, that for the `Deployment Examples` you'd need to add one or multiple test deployments in a `.test` sub-folder similar to the other modules. Please check the comment where I described the structure to get a feeling for what I mean.

Reply:
We usually try to keep the manual work to an absolutly minimum as it otherwise can become very tedious.

Reply:
I updated the readme using the PS utility. I still need to go back and add examples